includes/handler/class-inbox.php

```diff
@@ -56,9 +56,11 @@ public static function handle_inbox_requests( $data, $user_id, $type, $activity
 			 * @param array $object_types The object types to persist in the inbox.
 			 */
 			$object_types = \apply_filters( 'activitypub_persist_inbox_object_types', Base_Object::TYPES );
-			$object_types = \array_map( 'strtolower', $object_types );
+			$object_types = \array_map( 'Activitypub\camel_to_snake_case', $object_types );
 
-			if ( isset( $data['object']['type'] ) && ! \in_array( \strtolower( $data['object']['type'] ), $object_types, true ) ) {
+			if ( ! is_array( $data['object'] ) ) {
+				$success = true;
```

Comment:
At this point `$success` is already true and `$object_types` are unused. Maybe line 52 should be 
```php
// A comment that explains what's being tested for.
if ( $success && is_array( $data['object'] ) ) {
```